.vib/keycloak/vib-verify.json

```diff
@@ -22,7 +22,7 @@
           "url": "{SHA_ARCHIVE}",
           "path": "/bitnami/keycloak"
         },
-        "runtime_parameters": "ImF1dGgiOgogICJhZG1pblBhc3N3b3JkIjogIlMzOUJLV2pTa2giCiAgIm1hbmFnZW1lbnRQYXNzd29yZCI6ICJTMzlCS1dqU2toIg==",
+        "runtime_parameters": "YXV0aDoKICBhZG1pblBhc3N3b3JkOiBDb21wbGljYXRlZFBhc3N3b3JkMTIzNCEKICBtYW5hZ2VtZW50UGFzc3dvcmQ6IENvbXBsaWNhdGVkTWFuYWdlbWVudFBhc3N3b3JkMTI0NCE=",
```

Comment:
I saw you are not adding service-related params. In most pipelines, we are setting the service's port and type to `80` and `LoadBalancer` even if they use those values by default.

I think having them always in `runtime_parameters` is useful to better debug an issue if the default chart values are changed, but is this something we should have as a best practice or something of little importance?

Reply:
I agree this is convenient for debugging, so I'll make this change. 
I saw the importance of this in recent test failure investigation for Kibana.